pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -30,13 +30,23 @@ let
     # set to false if you want to control where to save the generated config
     # (e.g., in ~/.config/init.vim or project/.nvimrc)
     , wrapRc ? true
-    , neovimRcContent ? ""
+    , neovimRcContent ? null
+    , neovimLuaRcContent ? null
```

Comment:
`neovimLuaRcContent ? ""` makes more sense because we want to call nvim with `-u` even if neovimLuaRcContent is empty because it makes neovim more deterministic by ignoring user config when `wrapRc` is true.

Reply:
some bikeshedding: the `neovim` prefix in `neovimLuaRcContent` looks redundant since it's a neovim argument already. I know it's consistent with my previous choice neovimRcContent and that I am basically critisizing myself but normally few users would have used it, it's not documented so we could imagine deprecating "neovimRcContent" (I would do it in another PR) so dont let that stop you from choosing another name. at first I had though `initLua` but this is not the full initLua, parts of it is going to be generated + it might source the viml. I guess `luaRcContent` would be fine ? what do you think ?